main/autobot/project_tracker.js

```diff
@@ -0,0 +1,37 @@
+var Handler = require('../autobot/handler');
+var JiraApi = require('jira').JiraApi;
+
+var JiraTracker = function (config) {
+    return new JiraApi('https',
+        config.host,
+        config.port,
+        config.user,
+        config.password,
+        'latest');
+}
+
+var ProjectTracker = function (name) {
+    this.tracker = JiraTracker({
+        host: 'lumoslabs.atlassian.net',
+        port: null,
+        user: 'alan@lumoslabs.com',
+        password: ''
+    });
+}
+
+ProjectTracker.prototype = {
+    getStory: function (storyId, callback) {
+        var handler = new Handler(callback);
```

Comment:
I didn't get the idea of the status callback. I imagine we can always pass around the autobot standard `Handler`.